config.py

```diff
@@ -59,15 +59,35 @@ class BaseConfig:
         'es': '@'.join(['redaccion', 'cafebabel.com']),
         'it': '@'.join(['redazione', 'cafebabel.com']),
     }
-    ARTICLES_IMAGES_URL = '/static/uploads/articles'
+
+    DOMAIN_LANGUAGES = {
```

Comment:
Does it override `SERVER_NAME`? (in use for instance with `url_for` + `_external`) How do we test that?

Reply:
Which makes me think that if we want to be able to link translations across domains we will have to deal with a custom `url_for`. Oh and we need to discuss how we store/retrieve uploads across languages too! ;)

Reply:
It doesn't override `SERVER_NAME`. It is a mapping for discovering what is the language based on the domain name.